docs/recipes/package.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I still prefer if we put everything in `docs/recipes/gulp/` so this example is standalone. When we have more recipes in the future, it'll be less confusing on what npm dependencies, scripts, etc. are required for each.